### PR TITLE
1195 EC with CQ orgs by prio

### PR DESCRIPTION
**Pull request by leite08**
Ref: https://github.com/metriport/metriport-internal/issues/1195

### Dependencies

- upstream: none
- downstream: https://github.com/metriport/metriport/pull/1216

### Description

- EC by CQ org prio
- org created w/ high CQ orgs
- script to rebuild the CQ org list
- get-org retuns undefined
- don't notify on missing WH settings
- settings using cxId instead of id
- indicate final test result when running on monorepo
- remove alerts from DAPI lambdas

### Release Plan

- nothing special

### CODE REVIEW

File: packages/api/src/command/medical/organization/get-organization.ts [hunk 1]
```diff
@@ -3,11 +3,14 @@ import { OrganizationModel } from "../../../models/medical/organization";
 
 type Filter = Pick<OrganizationModel, "cxId"> & Partial<Pick<OrganizationModel, "id">>;
 
-export const getOrganization = async ({ cxId, id }: Filter): Promise<OrganizationModel | null> => {
+export const getOrganization = async ({
+  cxId,
+  id,
+}: Filter): Promise<OrganizationModel | undefined> => {
```
Comment (leite08) [inline packages/api/src/command/medical/organization/get-organization.ts:9]: Just to move towards the pattern of not using `null` inside our code

File: packages/api/src/command/webhook/webhook.ts [hunk 2]
```diff
@@ -46,18 +46,10 @@ async function missingWHSettings(
...
   log(msg + ` (url: ${webhookUrl}, key: ${loggableKey}`);
-  capture.message(msg, {
-    extra: {
-      webhookRequestId: webhookRequest.id,
-      webhookUrl,
-      webhookKey: loggableKey,
-      context: `webhook.processRequest`,
-    },
-    level: "info",
-  });
```
Comment (leite08) [inline packages/api/src/command/webhook/webhook.ts:60]: Not reporting missing WH settings to Sentry/Slack any longer

File: packages/core/src/external/commonwell/cq-bridge/coverage-enhancer.ts [hunk 3]
```diff
@@ -21,4 +45,75 @@ export abstract class CoverageEnhancer {
...
+    // Definition of priority of orgs to be included in the CQ orgs list.
+    // The order here matters!
+    const orgRetrievers = [
+      this.getMedium(orgs, states, "Epic"),
+      this.getMedium(orgs, states),
+      this.getMedium(orgs),
+      this.getLow(orgs, states, "Kno2"),
+      this.getLow(orgs, states),
+      this.getLow(orgs),
+    ];
```
Comment (leite08) [inline packages/core/src/external/commonwell/cq-bridge/coverage-enhancer.ts:79]: The core of the logic to determine the priority of CQ orgs for EC.

File: packages/api/src/command/settings/updateSettings.ts [hunk 4]
```diff
@@ -31,12 +31,12 @@ export const updateSettings = async ({
 };
 
 export type UpdateWebhookStatusCommand = {
```
Comment (jonahkaye) [inline packages/api/src/command/settings/updateSettings.ts:33]: what is the motiviation behind adding cxId to updateWebhook status?

File: packages/api/src/command/settings/updateSettings.ts [hunk 5]
```diff
@@ -31,12 +31,12 @@ export const updateSettings = async ({
...
-  id: string;
+  cxId: string;
   webhookEnabled: boolean;
   webhookStatusDetail?: string;
 };
 export const updateWebhookStatus = async ({
-  id,
+  cxId,
```
Comment (leite08) [inline packages/api/src/command/settings/updateSettings.ts:39]: As I was reviewing the WH code to remove the notification I remembered how it was confusing for someone's onboarding (I think yours!) WRT settings having an `id` column but that actually meaning the cx's ID. So I wend on and updated the parameters on the functions to better represent what that field really means.

File: packages/utils/src/commonwell/cq-org-builder.ts [hunk 6]
```diff
@@ -0,0 +1,176 @@
...
+  console.log(`Done at ${new Date().toISOString()}`);
+}
+
+async function getInfo(url: string) {
+  return retryFunction(() => axios.get(url), 5, 500);
+}
+
+async function getOrgStatesFromSequoia(orgOID: string): Promise<string[]> {
+  if (!orgOID) return [];
+  const url =
```
Comment (RamilGaripov) [inline packages/utils/src/commonwell/cq-org-builder.ts:118]: nit: use URLSearchParams instead? :P

File: packages/utils/src/commonwell/cq-org-builder.ts [hunk 7]
```diff
@@ -0,0 +1,176 @@
...
+    `${sequoiaQueryURL}` +
+    `?_format=json` +
+    `&_id=${orgOID}` +
+    `&_radius=30` +
```
Comment (RamilGaripov) [inline packages/utils/src/commonwell/cq-org-builder.ts:122]: nit: `30` is kinda hidden here. Maybe put this into a const

File: packages/utils/src/commonwell/cq-org-builder.ts [hunk 8]
```diff
@@ -0,0 +1,176 @@
...
+}
+
+function getPrio(org: OrgWithGateway): OrgPrio {
+  if (priorityOrgs.high.includes(org.id)) return "high";
+  if (priorityOrgs.medium.includes(org.id)) return "medium";
+  return "low";
+}
+
+// from API
+const retryFunction = async <K>(fn: () => Promise<K>, maxRetries = 3, waitTime = 3000) => {
```
Comment (RamilGaripov) [inline packages/utils/src/commonwell/cq-org-builder.ts:158]: seems like this could be useful to have on some shared location

**Review by RamilGaripov — APPROVED** [review]
Looks like Christmas came early - this PR is like a grab bag of goodies 👍 

Approved with a couple of nits

**Review by Orta21 — APPROVED** [review]

**Review by Orta21 — APPROVED** [review]

File: packages/utils/src/commonwell/cq-org-builder.ts (hunk 6 above)
Comment (leite08) [reply packages/utils/src/commonwell/cq-org-builder.ts:118]: yeah, I just copy/paste this code and did some minor adjustments, but I agree. Updating 👍

File: packages/utils/src/commonwell/cq-org-builder.ts (hunk 8 above)
Comment (leite08) [reply packages/utils/src/commonwell/cq-org-builder.ts]: yeah, this comes from the API, moved to shared

**Pull request merged**